pyvista/core/dataset.py

```diff
@@ -165,32 +215,102 @@ def active_tensors_name(self) -> str:
 
     @active_tensors_name.setter
     def active_tensors_name(self, name: str):
-        """Set the name of the active tensor."""
         self.set_active_tensors(name)
 
     @property
     def active_vectors_name(self) -> str:
-        """Return the name of the active vectors array."""
+        """Return the name of the active vectors array.
+
+        Examples
+        --------
+        Create a mesh, compute the normals, and return the
+        name of the active vectors.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh_w_normals = mesh.compute_normals()
+        >>> mesh_w_normals.active_vectors_name
+        'Normals'
+
+        """
         return self.active_vectors_info.name
 
     @active_vectors_name.setter
     def active_vectors_name(self, name: str):
-        """Set the name of the active vector."""
         self.set_active_vectors(name)
 
     @property
     def active_scalars_name(self) -> str:
-        """Return the active scalar's name."""
+        """Return the name of the active scalars.
+
+        Examples
+        --------
+        Create a mesh, add scalars to the mesh, and return the name of
+        the active scalars.
+
+        >>> import pyvista
```

Comment:
We usually put setting examples in the getter's docstring, since that's only what is accessible and what get rendered online. See e.g. https://github.com/pyvista/pyvista/blob/1e8a2b84afe9db03ea9a658a3eababae52c29a64/pyvista/plotting/lights.py#L722-L759 and most other `Light` property docstrings.